mirage/scenarios/dashboard.ts

```diff
@@ -57,6 +58,30 @@ export function dashboardScenario(server: Server, currentUser: ModelInstance<Use
         index: 0,
     });
 
+    // Addons for filesNode
+    const dropbox = server.schema.addons.find('dropbox') as ModelInstance<AddonModel>;
+    const dropboxAccount = server.create('external-account', {
+        displayName: 'Bugs Bunny',
+        provider: dropbox,
+    });
+    const dropboxAccountTwo = server.create('external-account', {
+        displayName: 'Daffy Duck',
+        provider: dropbox,
+    });
+    server.create('user-addon', {
+        id: 'dropbox',
+        externalAccounts: [ dropboxAccount, dropboxAccountTwo ],
+        userHasAuth: true,
+        user: currentUser,
```

Comment:
I think we could add `addon: dropbox,` here. Not sure if we'll need it, but may save us a bit of time later